docs/InfoListItem.md

```diff
@@ -0,0 +1,42 @@
+# InfoListItem
+The InfoListItem is intended to be used in List views. It positions a title as well as optional subtitle(s), icon, and status stripe.
+
+<img width="100%" alt="Info List Items in a variety of styles" src="./images/infoListItem.png">
+
+## Usage
+
+```typescript
+// app.module.ts
+import { InfoListItemModule } from '@pxblue/angular-components';
+...
+imports: [
+    InfoListItemModule
+  ],
+```
+```typescript
+<pxb-info-list-item title="Status" divider="full" [statusColor]="colors.green[500]">
+    <mat-icon [style.color]="colors.green[500]" icon>eco</mat-icon>
+</pxb-info-list-item>
+```
+
+## API
+
+<div style="overflow: auto;">
+
+| Prop Name         | Description                                      | Type                                               | Required | Default        |
```

Comment:
I believe we're calling this column attributes for angular components now

Reply:
You're correct, updated this.